### ISSUE TITLE
Netdata install failure on OMV5 on Raspberyy-Pi4

### ISSUE THREAD

**Issue by Copaxy** [netdata/netdata#9344]
##### Question summary
Hello, 
it is my first time using github and i never did a issue before but i hope i do nothing wrong. 

 I think my issue is so easy for you and the pros probbably rolling their eyes but still please help me 😅
I have a failure while installing netdata on OMV5 on my Raspberry-Pi 4 with portainer. 
Because i am just learning doing stuff with dockers i searched for a tutorial but all tutorials i found were for old versions and gave me errors or didn't work. I tried to do a own container but it failed.  
So i tried this code for a Stack in portainer from someone else: 

```
version: '2'
services:
  netdata:
    image: netdata/netdata
    ports:
      - 19999:19999
    cap_add:
      - SYS_PTRACE
    security_opt:
      - apparmor:unconfined
    volumes:
      - /proc:/host/proc:ro
      - /sys:/host/sys:ro
      - /var/run/docker.sock:/var/run/docker.sock:ro

```

But i get this error: 
 
![image](https://user-images.githubusercontent.com/66993806/84750485-a0382280-afbb-11ea-821f-850c978e2b3e.png)

(I hope the picture works) 

Deployment error, Error response from daemon: Get https://registry-1.docker.io/v2/:dial tcp: lookup registry-1.docker.io on [::1]:53:read udp [::1]:53464->[::1]:53: read: connection refused

what does that mean, What do i wrong? and can you recommend me a good tutorial or guide for learning to do stuff in portainer which is up to date?

I am a newbi, i just two days ago got into docker and portainer, so sorry for stupidy

**Comment by prologic** [comment]
@Copaxy Hello! 👋Welcome to Netdata! Let me help you out here :) (_we all a n00bs! don't worry!_).

> Deployment error, Error response from daemon: Get https://registry-1.docker.io/v2/:dial tcp: lookup registry-1.docker.io on [::1]:53:read udp [::1]:53464->[::1]:53: read: connection refused

This error looks very much like DNS failure. Portioner (_however you've set this up_) and the Docker engine/daemon on your RPi4 is unable to lookup a valid IP address for the Docker Regsitry at `registry-1.docker.io`.

I would suggest firstly you fix this on your RPi4 and ensure you can:

- Perform hostname lookups, e.g: `host google.com`
- Can connect to an internet address: `e.g: telnet google.com 80`

Then try again :)

**Comment by Copaxy** [comment]
Hi thanks for your answer! :) 

Yes i will do this and come back for questions :) 

Many thanks for your reply!

**Comment by Copaxy** [comment]
Hi i have good news. I fixed the problem :) 

> This error looks very much like DNS failure. Portioner (however you've set this up) and the Docker engine/daemon on your RPi4 is unable to lookup a valid IP address for the Docker Regsitry at registry-1.docker.io.

Thanks to prologic i got an idea what was going on. I first tried host and ping a DNS Server to check if there is internet access. This was succesful, internet access was there. But there was no ethernet and wifi shown up in the network settings. So i reinstalled OMV again to make sure everything is right. 
- First i installed debian buster and set up my SATA-HAT-Board with the needed software. And checked if everything works-> `systemctl status rockpi-sata.service`

- Second i did the whole reset password and ssh thing for remote access on putty. 

- Next i directly installed OMV via terminal without doing any settings on my wifi and ethernet to make sure nothing is influencing OMV. 

- Next i now installed the driver for my realtek WIFI-Stick. -> i Have this one [https://www.amazon.de/dp/B077X5Y2LT?tag=sternvgl03-21](url) i have this one for backup if ethernet does somehow is bugging

- Next i set up OMV -> admin password 

- Then set up SFTP and did my port settings on router 

- Then finally after everything before worked and also finally showed up on network settings, i set up the netdata stack with the same code from above because i am still not so skilled to set it up manually in a container. -> And it worked!


I think this was my problem: previously i installed the driver for my wifi stick before installing OMV and also i did disable the wifi0 (on board) because i wanted to use directly the wifi1 (my wifistick)

I did disable wifi0 with blacklist on raspberry-pi with the lines
 `blacklist brcmfmac
blacklist brcmutil` 

Maybe these two things did problems on OMV so thats why internet worked but portainer (docker didn't) -> But who knows

But it works now :) 

Thank you for helping 👍
